Metallum/Classes/Controllers/BandsViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  BandsViewController.swift
+//  Metallum
+//
+//  Created by AppleTree on 3/8/16.
+//  Copyright © 2016 OlhaF. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class BandsViewController: UITableViewController {
+
+    var bands: [Band]?
```

Comment:
you can initialise this variable with empty array and do not declare it as optional. So you don't have to do unnecessary checks on optional